htdocs/sass/rcloud-discover.scss

```diff
@@ -99,6 +99,10 @@ progress::-webkit-progress-value {
 .grid-item h1{
     font-size: 17px;
     margin: 0;
+    /* firefox */
+    white-space: pre-wrap;
+    word-break: break-all;
+    /* chrome */
     word-break: break-word;
```

Comment:
Maybe the issue is that this value doesn't exist? It's a `word-wrap` value instead. Confusing.